extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -235,7 +277,10 @@ public boolean equals(Object o)
            Objects.equals(basePersistDirectory, that.basePersistDirectory) &&
            Objects.equals(indexSpec, that.indexSpec) &&
            Objects.equals(segmentWriteOutMediumFactory, that.segmentWriteOutMediumFactory) &&
-           Objects.equals(intermediateHandoffPeriod, that.intermediateHandoffPeriod);
+           Objects.equals(intermediateHandoffPeriod, that.intermediateHandoffPeriod) &&
+           Objects.equals(logParseExceptions, that.logParseExceptions) &&
+           Objects.equals(maxParseExceptions, that.maxParseExceptions) &&
+           Objects.equals(maxSavedParseExceptions, that.maxSavedParseExceptions);
```

Comment:
New variables can't be null.

Reply:
Changed these, I don't really think this matters much though since these comparisons aren't in any hot path